setup.cfg

```diff
@@ -33,7 +33,10 @@ include_package_data = True
 setup_requires =
   setuptools_scm
 install_requires =
-  sunpy[net]
+  sunpy[net]>=2.1.1
+  numpy<1.20.0
```

Comment:
Why this?

Reply:
Oh I was getting weird build error similar to 'Expected 216 from C header, got 192 from PyObject.' and this was the only way I could get it to work I can remove and try again.

Reply:
Sounds like something we should either ignore or workout why and fix.

Reply:
yea so this happen https://dev.azure.com/sunpy/radiospectra/_build/results?buildId=14746&view=logs&j=0609049e-3a2a-5217-995e-2cc283dcaa52&t=93461a68-a5a2-5d84-2961-2a4e1719748a&l=803